### PR TITLE
⚡️ Add blog menu page

### PR DESCRIPTION
**Pull request by feliciofilipe**
Adds a blog to the landing page.

Depends on the pr #38 for the Dark Mode to work.

### CODE REVIEW

**Review by nelsonmestevao — CHANGES_REQUESTED** [review]
I like it very much :clap: 

The blog posts should be in markdown files under some `posts` or `_posts` folder (there's even an example where we could write `.mdx` inside `pages/posts`). We could write those in markdown files and have all metadata inside the YAML front matter.

Next.js has several examples on how to achieve this with static generation. Checkout this list:

- [A statically generated blog example using Next.js, Markdown, and TypeScript](https://github.com/vercel/next.js/blob/canary/examples/blog-starter-typescript/README.md)
- [Portfolio Starter Kit](https://github.com/vercel/next.js/blob/canary/examples/blog/README.md)

File: landing/blog/Entry/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            href="#"
+            class="text-2xl font-bold text-dark hover:text-primary hover:underline"
+          >
+            {title}
+          </a>
+          <p class="mt-2 text-gray-600">{text}</p>
+        </div>
+        <div class="flex items-center justify-between mt-4">
+          <a href="#" class="text-primary hover:underline">
+            Read more
```
Comment (nelsonmestevao) [inline landing/blog/Entry/index.js:22]:
```suggestion
            Ler mais
```

File: pages/blog.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    photo="filipe.png"
+                    numberPosts={2}
+                  />
+                  <Author
+                    author="Nelson Estevão"
+                    photo="nelson.png"
+                    numberPosts={2}
+                  />
+                  <Author
+                    author="Marta Silva"
```
Comment (nelsonmestevao) [inline pages/blog.js:71]: I think name would be a better prop here. What do you think?

Comment (feliciofilipe) [reply pages/blog.js:71]: Yeah! Why not :yin_yang:

File: landing/index.js [hunk 3]
```diff
@@ -1,5 +1,10 @@
 import Footer from "./Footer";
 import Header from "./Header";
 import Hero from "./Hero";
+import Entry from "./blog/Entry";
```
[the remark this answers is not in the document]
Comment (feliciofilipe) [reply landing/index.js:4]: Yeah! Will do :yin_yang:

**Review by nelsonmestevao — APPROVED** [review]

**Pull request merged**